src/Miew.js

```diff
@@ -889,6 +919,40 @@ Miew.prototype._getBSphereRadius = function () {
   return radius * this._objectControls.getScale();
 };
 
+Miew.prototype._getbBox = function (matrix) {
```

Comment:
Here I have got some questions for the function name. 
I think it should be _getOBB, also we should mention in comments that this is OBB for the whole scene consisted of several molecules and current usage of that OBB.